src/scripts/main.js

```diff
@@ -2,10 +2,20 @@
 
 function waitFor(element, eventName) {
   // write your code here
+  return new Promise((resolve) => {
+    element.addEventListener(eventName, () => {
+      // eslint-disable-next-line
+      resolve(`It was ${eventName} on the element: ${element.nodeName}, id: ${element.id}.`);
+    });
+  });
 }
 
 const printMessage = (message) => {
   // write your code here
+  document.body.insertAdjacentHTML('beforeend',
+    `<div class="message">
+      ${message}
+    </div>`);
```

Comment:
```suggestion
  document.body.insertAdjacentHTML(
    'beforeend',
    `<div class="message">${message}</div>`
  );
```
I think it's clearer that way